app/models/loan_response.rb

```diff
@@ -38,7 +40,14 @@ def hash_data
   end
 
   def embeddable_media
-    embeddable_media_id.present? ? EmbeddableMedia.find(embeddable_media_id) : nil
+    if embeddable_media_id.present?
+      result = EmbeddableMedia.find_safe(embeddable_media_id)
+    end
+    unless result
+      result = EmbeddableMedia.where(owner_id: @owner.id, owner_type: @owner.class.name,
+        owner_attribute: @custom_field.id).first
+    end
+    result
```

Comment:
I'm not understanding what's going on here. It seems hacky. Why do we need two ways to associate an embeddable media with a loan response?